integration/client-data-test.ts

```diff
@@ -927,14 +927,9 @@ test.describe("Client Data", () => {
                   // Ignore any dev tools messages. This may only happen locally when dev
                   // tools is installed and not in CI but either way we don't care
                   /Download the React DevTools/.test(text) ||
-                  (templateName === "rsc-parcel-framework" &&
+                  (templateName.includes("rsc") &&
```

Comment:
While I was here, I also future proofed this against upcoming RSC Framework Mode work.